text/0079-cdk-2.0.md

```diff
@@ -0,0 +1,337 @@
+---
+feature name: cdk-2.0
+start date: 2020-3-29
+rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/156
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/6
+---
+
+<!-- replace the blockquoted sections with your content -->
+# Summary
+
+CDK v2.0!
+
+This RFC details the strategy for building and releasing aws-cdk 2.0, in addition to changes to tooling required to support this strategy. For details on the specific features themselves, see the corresponding RFCs.
+
+# Motivation
+
+Since the CDK was announcned as "generally available", the team has tried to limit breaking changes for users. However, a handful of features have emerged that the core team believes will make the CDK significantly simpler to use. These changes require a 2.0 release to signal to users that code changes may be required for adoption.
+
+The main change of note is to consolidate all of the aws service construct libraries into a single package. The motivation for this change is detailed in [the corresponding RFC](https://github.com/aws/aws-cdk-rfcs/issues/6).
```

Comment:
```suggestion
The main change of note is to release the CDK as a single package instead of multiple package. The motivation for this change is detailed in [the corresponding RFC](https://github.com/aws/aws-cdk-rfcs/issues/6).
```